### PR TITLE
Updated the package

### PR DESCRIPTION
**Pull request by drexljo**
--First on: Beware https://issues.folio.org/browse/OKAPI-857, as this is breaking the changes introduced herein.--
SOLVED with https://github.com/folio-org/okapi-debian/pull/1/commits/90c665b5f89dab0fe5da49c47aabb5e4ac727835

Files are imported from a hands-on Debian package build process, so there might be a little work needed to adapt them to the Git autobuild mechanism. 

I did not explicitely focus on upgrade functionality, although some tests indicated it works well.

For a hopefully complete changelog visit debian/changelog

### CODE REVIEW

**Comment by julianladisch** [comment]
Thank you for your contribution!
Do you have plans how to adjust https://github.com/folio-org/folio-ansible/tree/master/roles/okapi to make it work with both the old and the new okapi-debian package?

**Comment by drexljo** [comment]
As of now I try to get rid of all that is not folio itself and the underlying OS, so that means I'm planning to do a Debian metapackage folio-full, that contains all modules within Debian packages and takes care of updating/install all by itself, so ansible would not be necessary any more. Still way to go, but Okapi was a big step forward and a lot of effort can be recycled for modules (database installer, Debian specific files...).

**Comment by julianladisch** [comment]
A Debian metapackage is a good idea and should go into a separate repository, for example folio-debian.
This https://github.com/folio-org/okapi-debian package is used for https://github.com/folio-org/folio-ansible that is highly configurable:
https://github.com/folio-org/folio-ansible/blob/master/doc/ansible-variables.md
https://github.com/folio-org/folio-ansible/tree/master/group_vars
The continuous integration pipeline automatically builds and deploys the reference environments:
https://dev.folio.org/guides/automation/#reference-environments
Some institutions use the public FOLIO ansible playbook as a basis and extend it with local adjustments.
Ansible is used for provisioning, configuration management, deployment, continuous delivery and orchestration for multiple FOLIO servers. This cannot be replaced by Debian packages.
If this pull request is not intended to be used by folio-ansible it should go into a separate branch or separate repository, with README.Debian.md explaining the intended use cases.

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,19 @@
+folio-okapi (3.1.0-1) xenial; urgency=medium
+
+  * Imported Upstream version 3.1.0
```
Comment (julianladisch) [inline debian/changelog:3]: This line should be removed. Importing upstream versions should go via the https://github.com/folio-org/okapi-debian/tree/debian-upstream branch.

**Comment by drexljo** [comment]
I know what Ansible is, I use it for years now. There is the possibility to circumvent the manual interaction by just creating the file /etc/folio/okapi/okapi.conf beforehand, there is a check for that (debian/postinst:65). In the end, unattended-upgrades has to work with it too.
So while it is not MEANT to be used with ansible on our side, it is entirely possible.

**Comment by drexljo** [comment]
As for folio-full, I will first do that local, test and afterwards provide it. And since unattended-upgrades have to work with that too, there will also be a non-interactive route, although postgres database/user creation will not be possible with that (because no program gets superuser rights on a server, for this is 1st lesson in security class and basically a nobrainer).

**Comment by drexljo** [comment]
Update on https://issues.folio.org/browse/OKAPI-857:

Further examination revealed it is not breaking on new systems or after a reboot, since it's an issue with directories in /tmp.

**Comment by dcrossleyau** [comment]
Closed.
The okapi-debian repo has moved. See README.

**Pull request closed without merging**